### DISCUSSION TITLE
Does not accept change in the lens.

### DISCUSSION THREAD

**Discussion by cecarpioch** [gopro/labs#1132] [Q&A]
Good afternoon, I would like to know if anyone has a solution to why it won't let me change the lens to linear in 4k and 5.3k parameters at 24 or 30 FPS for the Hero 12 (v2.20.70). Whenever I set the lens type to linear or linear plus locked horizon... then I scan the qr, it stays in wide.

Has this happened to anyone else? Thanks!

**Comment by dnewman-gpsw** [comment]
The command fL doesn't work?

**Reply by cecarpioch** [reply]
![WhatsApp Image 2024-10-19 at 7 17 25 PM](https://github.com/user-attachments/assets/b29b7b56-75cb-4589-a222-bf4ca26cad52)

**Comment by cecarpioch** [comment]
![WhatsApp Image 2024-10-19 at 7 18 54 PM](https://github.com/user-attachments/assets/25deebfc-eac3-4d8c-9a70-d54d8c35d533)

**Reply by dnewman-gpsw** [reply]
Your camera doesn't have 4x3 4K linear.  Only ask for modes your camera has.
